templates/README.md.mustache

```diff
@@ -83,4 +82,23 @@ make install
 After installation, the included modules are available under
 the `{{ namespace }}` namespace.
 
+{{# extracted }}
+## {{& extracted_fullname }}
+
+{{& extracted_description}}
+
+{{# supported_ocaml_versions }}
+- Compatible OCaml versions: {{& text }}
+{{/ supported_ocaml_versions }}
+- Additional OCaml dependencies:{{# extracted_dependencies }}
+  - {{& description }} {{/ extracted_dependencies }}{{^ extracted_dependencies }} none{{/ extracted_dependencies }}
+
+{{& extracted_fullname }} can be installed via OPAM as above using the package name `coq-{{ extracted_shortname }}`.
+
+To instead extract the code and compile manually, use the following command:
+```
+make {{& target }}
```

Comment:
Consider `extracted_target` here, and `target` for library build?
https://github.com/coq-community/manifesto/blob/8cbc838800c0a00ce3d638d63728b1f3bf7c730f/templates/README.md.mustache#L78